docs/quickstart.md

```diff
@@ -1,70 +1,163 @@
+You can monitor the progress using `kubectl`:
 
 ```sh
-$ redskyctl generate trial --interactive -f <(kubectl get experiment postgres-example -o yaml)
+$ kubectl get trials -o wide -w
+```
+You should see output similar to this:
+```sh
+NAME                        STATUS      ASSIGNMENTS             VALUES
+postgres-example-000   Completed   cpu=1319, memory=1457   duration=7, cost=33
+postgres-example-001   Completed   cpu=963, memory=2647    duration=5, cost=29
+postgres-example-002   Completed   cpu=2394, memory=1934   duration=5, cost=58
+postgres-example-003   Completed   cpu=3508, memory=2654   duration=6, cost=85
+postgres-example-004   Completed   cpu=3410, memory=1019   duration=5, cost=78
+postgres-example-005   Completed   cpu=2757, memory=2538   duration=4, cost=68
+postgres-example-006   Completed   cpu=983, memory=3057    duration=6, cost=30
+postgres-example-007   Completed   cpu=373, memory=3065    duration=15, cost=17
+postgres-example-008   Waiting     cpu=1198, memory=2701
 ```
 
-You can monitor the progress using `kubectl`:
+The trials will run until the `experimentBudget` is satisfied. In this example, there will be 40 trials that run.
+While the trial is running, there may be assignment combinations that are unstable and result in a failure.
+
+After the trial is complete, you will be able to view the parameters and the metrics generated from the trial.
+The results can be reviewed as a visualization by running the following command:
 
 ```sh
-$ kubectl get trials
+$ redskyctl results
 ```
 
-When running interactive trials in a single namespace, be sure only trial is active at a time.
+This should open a link in your browser showing a graph similar to the below.
+![redskyctl results](images/redskyresults.png)
+
+This is where you can review the trials that have taken place and decide which parameter makes the most appropriate trade-off.
+There is a filter on the upper right hand side where you can narrow your results to show only the `best` trials.
+These are the trials with the best possible combination of cost and duration, meaning if you hold cost fixed at a given value, we have found the best possible duration.
+You can choose any `best` configuration that works for your cost and duration preferences.
+Often you find that since the metrics are not linearly related, there are some points that give a bigger boost in duration at a similar cost, so it is useful to explore all of the options to find the best one for you.
 
-After the trial is complete, you will be able to view the parameters and the metrics generated from the trial. The metrics can be used to gauge how effective the used parameters were.
+Congratulations! You just ran your first experiment.
+Now that you've got an idea of the basics, you can learn more about the experiment lifecycle [here][lifecycle] or learn more about experiment concepts [here][concepts].
```

Comment:
I'm going on the record as not being a huge fan of "here" link text, e.g. instead of `lifecycle [here][lifecycle]` just use `[lifecycle](lifecycle.md)` (or `[lifecycle][]` if the reference is fixed).